frontend/admin/src/js/components/user/UserTableFilterDialog.tsx

```diff
@@ -257,18 +257,21 @@ export type UserTableFiltersProps = Pick<
   "onSubmit" | "enableEducationType"
 > & {
   isOpenDefault?: boolean;
+  initialActiveFilters?: FormValues;
 };
 
 const UserTableFilters = ({
   onSubmit,
   isOpenDefault = false,
   enableEducationType,
+  initialActiveFilters,
   ...rest
 }: UserTableFiltersProps) => {
   const [isOpen, setOpen] = React.useState<boolean>(isOpenDefault);
   const { emptyFormValues } = useFilterOptions(enableEducationType);
-  const [activeFilters, setActiveFilters] =
-    useState<FormValues>(emptyFormValues);
+  const [activeFilters, setActiveFilters] = useState<FormValues>(
+    initialActiveFilters ?? emptyFormValues,
+  );
```

Comment:
Just to hint at "initialState", which is var name in useState docs that we pass in:

```suggestion
  const initialFiltersState = activeFilters ?? emptyFormValues;
  const [activeFilters, setActiveFilters] = useState<FormValues>(initialFiltersState);
```